src/en/space-station-14/character-species/guidelines.md

```diff
@@ -1,5 +1,174 @@
+
+ It is not neccessary to fine-tune balance at point of proposal. "This species is better in cold climates and moves slowly in warm ones" is preferable to "this species takes 20% less cold damage and moves 20% slower when in environments above its own core temperature".
+
+ It is actively discouraged to "bolt-on" upsides and downsides that distract from the core theme and identity of the species, especially at time of proposal. A species' design is far more important, and far harder to fix, than its mechanics.
+
+ A species **must not** be intentionally designed in a way that makes them specifically good or bad at one particular job on the space station. For example, a short, bearded humanoid species fond of drink and industry should not be, by design, better at being a miner or salvager than any other member of the crew. An ability they have that is useful in those roles should also be useful in other roles.
+
+ It is acceptable for a species to be *emergently* better at some aspect of a job. For example, a species with a tail that can pull things might be better than average at moving crates around the station, which is valuable to a Cargo team. But this is a skill that is useful broadly, rather than being focussed on one particular discipline.
+
+ A species **must not**, under **any** circumstances, be designed in a way that makes them unusually good or unusually bad at combat. Combat is an essential part of Space Station 14, and species with wildly divergent abilities make the act of balancing it extremely difficult.
+
+ ### Rule 4: Speciesism
+
+ Speciesism is banned on Wizard's Den in its entirety. As such, species **must not** directly encourage the "othering" of those that play those species - for example, causing health problems in those around them, or being incapable of socializing or interacting healthily with the rest of the crew.
+
+ ### Rule 5: Deliberately Challenging Species
+
+ Species **must** not be designed in such a way that a new player, picking a random character and rolling that species, would have an excessively negative experience due to lack of game knowledge. This **does not** mean a species cannot have a notable, potentially deadly, downside. However, that downside **should** be recoverable from, presented obviously to a player who has no prior context of that species, and teachable within the game by other players.
+
+ For example: a species suffers from oxygen poisoning. This means they must wear internals at all times, except in specific circumstances where oxygen is not present. This species **should not** punish the player by dying rapidly if their internals fail or are removed, and, if they do die, they **should** be revivable.
+
+ If a species is significantly challenging to play as, compared to normal round-start species, the following rules apply:
+ - It **must not** be able to be selected round-start by new players.
```

Comment:
I would say this can be allowed, but have guidebook open for that species when a new player selects it.

Reply:
![Uploading 7i6e3v-2589314892.jpg…]()